uk.co.compucorp.civicrm.tasksassignments/gulpfile.js

```diff
@@ -80,8 +80,8 @@ var test = (function () {
     for: function (srcFile) {
       var srcFileNoExt = path.basename(srcFile, path.extname(srcFile));
       var testFile = srcFile
-      .replace('src/tasks-assignments/', 'test/')
-      .replace(srcFileNoExt + '.js', srcFileNoExt + '.spec.js');
+        .replace('src/tasks-assignments/', 'test/')
```

Comment:
if you are not making any changes in this file (it seems they are all changes from the linter) then please remove it from the POC, it just adds noise